### PR TITLE
Add additional documentation for opt-out annotation

### PR DESCRIPTION
**Pull request by jpeeler**
This is a follow up to #716.

### CODE REVIEW

File: contributors/design-proposals/pod-preset.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
       * [Validations](#validations)
     * [AdmissionControl Plug-in: PodPreset](#admissioncontrol-plug-in-podpreset)
       * [Behavior](#behavior)
+      * [PodPresetOptOut Annotation](#podpreset-opt-out)
```
Comment (cblecker) [inline contributors/design-proposals/pod-preset.md]: `#podpresetoptout-annotation`

File: contributors/design-proposals/pod-preset.md [hunk 2]
```diff
@@ -192,7 +193,8 @@ injection. These are as follows:
 
 The **PodPreset** plug-in introspects all incoming pod creation
 requests and injects the pod based off a `Selector` with the desired
-attributes.
+attributes, except when the [PodPresetOptOut Annotation](#podpreset-opt-out)
```
Comment (cblecker) [inline contributors/design-proposals/pod-preset.md]: `#podpresetoptout-annotation`

File: contributors/design-proposals/pod-preset.md [hunk 3]
```diff
@@ -210,6 +212,10 @@ This will be first implemented as an AdmissionControl plug-in then can be
 converted to an Initializer once that is fully ready. The proposal for
 Initializers can be found at [kubernetes/community#132](https://github.com/kubernetes/community/pull/132).
 
+#### PodPresetOptOut Annotation
+There may be instances where you wish for a pod to not be altered by any pod
+preset mutations. For these events, one can add an annotation in the pod spec
+of the form: `podpreset.admission.kubernetes.io/PodPresetOptOut: "true"`.
```
Comment (derekwaynecarr) [inline contributors/design-proposals/pod-preset.md]: the annotation was changed in review to: "podpreset.admission.kubernetes.io/exclude"

Comment (jpeeler) [inline contributors/design-proposals/pod-preset.md]: forgetful mistake, fixing

**Pull request merged**